engine-primitives/engine-primitives/attributes.go

```diff
@@ -112,7 +112,8 @@ func (p *PayloadAttributes) Validate() error {
 		return ErrEmptyPrevRandao
 	}
 
-	if p.Withdrawals == nil && p.version >= version.Capella {
+	// For any fork version Capella onwards, withdrawals are required.
+	if p.version.IsGreaterThanOrEqualTo(version.Capella()) && p.Withdrawals == nil {
```

Comment:
this could be an `!version.isBefore(version.Capella())`

Reply:
done ^